internal/controller/ols_app_server_deployment.go

```diff
@@ -129,6 +136,31 @@ func (r *OLSConfigReconciler) generateOLSDeployment(cr *olsv1alpha1.OLSConfig) (
 	if dataCollectorEnabled {
 		volumes = append(volumes, olsUserDataVolume)
 	}
+
+	if extraCertsDefined {
+		configMapName := extraCertsCM.Name
+		for key:= range extraCertsCM.Data {
+			volumeName := OLSExtraCertsVolumeName + "-" + key
+			volume := corev1.Volume{
+				Name: volumeName,
+				VolumeSource: corev1.VolumeSource{
+					ConfigMap: &corev1.ConfigMapVolumeSource{
+						LocalObjectReference: corev1.LocalObjectReference{
+							Name: configMapName,
+						},
+						Items: []corev1.KeyToPath{
```

Comment:
This is probably wrong, I think I need to use the actual data that are stored in CM under the key.

Reply:
If we set all keys of configmap available in this volume, we can omit the `Items`.
Reference https://kubernetes.io/docs/concepts/storage/volumes/#configmap